src/lib.rs

```diff
@@ -44,15 +44,16 @@ fn default_build_command() -> process::Command {
 }
 
 impl Build {
-    pub fn execute(
-        self,
-        crate_name: &str,
-        static_dir_path: impl AsRef<Path>,
-        build_dir_path: impl AsRef<Path>,
-    ) -> Result<()> {
+    pub fn execute(self, crate_name: &str, static_dir_path: impl AsRef<Path>) -> Result<()> {
         log::trace!("Build: get package's metadata");
         let metadata = metadata();
 
+        let build_dir_path = if self.release {
+            metadata.target_directory.join("release/dist")
+        } else {
+            metadata.target_directory.join("debug/dist")
+        };
```

Comment:
1. You need to join the components separately. Otherwise I'm not sure it will work on windows

```suggestion
        let build_dir_path = if self.release {
            metadata.target_directory.join("release").join("dist")
        } else {
            metadata.target_directory.join("debug).join("dist")
        };
```
2. It would be better if this build dir path was configurable somehow by the user. It used to be the case with the argument to the function but having a default like you did here is handy.